src/PharChecker.php

```diff
@@ -31,7 +40,7 @@ public function checkFileValidity($filename)
                 throw $e;
             }
 
-            return $e->getMessage();
+            return 'Failed to create phar. '.$e->getMessage();
```

Comment:
it sounds like it `Failed to create phar file`, not `Failed to create Phar instance`

Reply:
updated